lib/workers/repository/onboarding/branch/check.ts

```diff
@@ -61,7 +61,21 @@ export async function isOnboarded(config: RenovateConfig): Promise<boolean> {
     logger.debug('Config file will be ignored');
     return true;
   }
+
   const cache = getCache();
+  const onboardingBranchCache = cache?.onboardingBranchCache;
+  // if onboarding cache is present and base branch has not been updated branch is not onboarded
+  if (
+    onboardingBranchCache &&
+    onboardingBranchCache.defaultBranchSha ===
+      getBranchCommit(config.defaultBranch!)! &&
```

Comment:
```suggestion
      getBranchCommit(config.defaultBranch!) &&
```
i don't think we need it for compare